src/lib/components/ItemAction.svelte

```diff
@@ -10,7 +10,7 @@
     </div>
     <slot />
   </div>
-  <div class="actions">
+  <div>
```

Comment:
you still need the `div` then?

Reply:
Yes. I want the spacing to be set by the container here. If I add a slot and the slot is two elements, the gap here will apply also between those two elements. Which is not what we want.